pod-security/mutate/set-run-as-non-root/set-run-as-non-root.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: set-run-as-non-root
+  annotations:
+    policies.kyverno.io/title: Set runAsNonRoot
+    policies.kyverno.io/category: Pod Security Standards (Mutate)
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kyverno-version: 1.6.0
+    kyverno.io/kubernetes-version: "1.22-1.30"
```

Comment:
Have you personally verified this works across that many versions? Again, this annotation holds what was tested not what "should" work.